settings/src/components/email-address.js

```diff
@@ -47,44 +43,43 @@ export default function EmailAddress() {
 		try {
 			await edit( { pending_email: '' } );
 			await save();
-		} catch( error ) {
-			alert( error.message );
+		} catch ( error ) {
+			setEmailError( error.message );
```

Comment:
I'm unsure whether this is an appropriate message to be displayed to a user or what it'll look like. Needs testing.

Reply:
Tested this by returning a 404 and it appears it'll handle it appropriately

![Screen Shot 2023-05-01 at 4 19 18 PM](https://user-images.githubusercontent.com/1017872/235406067-5725447b-06f9-4cbe-b752-feab1f3574d3.jpg)